parser-typechecker/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -328,6 +329,7 @@ isFailure o = case o of
   ListDependents{} -> False
   TermMissingType{} -> True
   DumpUnisonFileHashes _ x y z -> x == mempty && y == mempty && z == mempty
+  NamespaceEmpty _ -> False
```

Comment:
Oops... we might want to consider this a failure instead (`True`). I'm not totally sure what the implications of that are.